pkg/logs/sources/replaceable_source.go

```diff
@@ -33,6 +33,7 @@ func (r *ReplaceableSource) Replace(source *LogSource) {
 	r.Lock()
 	defer r.Unlock()
 	r.source = source
+	r.source.Status.Success()
```

Comment:
Should we instead be setting the new source status to the same as the old source status? 

ex: 
```
newSource.Status = r.source.Status
r.source = newSource
```
What if the old source had failed for some reason, and then the source gets replaced.